pom.xml

```diff
@@ -109,6 +109,8 @@
     <dep.swagger.version>1.3.12</dep.swagger.version>
     <dep.swagger-plugin.version>2.3.2</dep.swagger-plugin.version>
 
+    <dep.protobuf.version>3.3.1</dep.protobuf.version>
```

Comment:
We're currently on an older versions of protobuf and the upgrade process is nontrivial for large, distributed codebases.